drivers/virtualization/virt_ivshmem_shell.c

```diff
@@ -8,7 +8,7 @@
 #include <stdlib.h>
 #include <zephyr/drivers/virtualization/ivshmem.h>
 
-static const struct device *ivshmem;
```

Comment:
So it's not part of bss (which is reset to 0 at boot) or did gcc moved that out of this section? Used to work. 

Well, anyway, better the way you did.